src/models/transfers.js

```diff
@@ -434,10 +447,7 @@ function * setTransfer (transfer, requestingUser) {
   }
 }
 
-function * getFulfillment (transferId, requestingUser) {
-  if (!requestingUser || !requestingUser.name) {
```

Comment:
Ok, I think we were planning on minimizing the anonymous endpoints to mitigate the risk of denial of service attacks, but this should be fine for now